integration-manifest.json

```diff
@@ -6,6 +6,7 @@
   "support_level": "kf-supported",
   "update_catalog": true,
   "link_github": true,
+  "release_dir": "bin/Release/netcoreapp3.1",
```

Comment:
previously we were targeting bin/Release/net472/export since additional DLLs get copied to that directory which are required for the PAM Provider to work. this will need to target an export directory (which needs to get the manifest.json as well) and potentially contain 2 releases for version compatibility: .net framework 4.7.2 and netcore 3.1